tests/behat/features/insert-content-link.feature

```diff
@@ -8,7 +8,8 @@ Feature: Insert an internal link into content
     Given a "subsite" "Subsite B"
     And a "page" "My page" with "URLSegment"="my-page", "Content"="My page content"
     And a "page" "Another page" with "URLSegment"="another-page", "Content"="My other page content"
-    And I am logged in with "CMS_ACCESS_CMSMain" permissions
+    And the "group" "EDITOR" has permissions "Access to 'Pages' section"
+    And I am logged in as a member of "EDITOR" group
```

Comment:
```suggestion
    And I am logged in with "CMS_ACCESS_CMSMain" permissions
```
There's no need to create a group for just the one permission - `I am logged in with "CMS_ACCESS_CMSMain" permissions` is already correct here.

Reply:
DONE